web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -504,10 +504,9 @@ You can follow these steps to migrate to the new auth system (assuming you alrea
     // highlight-start
     export const fields = defineUserSignupFields({
       displayName: async (data) => {
-        if (!data.profile || !data.profile.displayName) {
-          throw new Error('Display name is not available')
-        }
-        return data.profile.displayName
```

Comment:
This was throwing TS error, so I refactored it to have casting to `any`.

Reply:
I have to admit, I wrote this out of my head and didn't test it. Good catch 👍

Reply:
Should we maybe mentioned Zod here ... or more ideally in the `userSignupFields` docs?